packages/dds/tree/src/test/shared-tree/tableTree.bench.ts

```diff
@@ -0,0 +1,722 @@
+					stackOperation: (undoRedoManager) => {
+						for (let i = 0; i < count; i++) {
+							// Redo insert row
+							undoRedoManager.redo();
+							// Redo insert column
+							undoRedoManager.redo();
+						}
+						assert(!undoRedoManager.canRedo);
+					},
+					maxBenchmarkDurationSeconds,
+				});
+			});
+
+			describe(`Insert a column and a row and remove right away`, () => {
+				// Test the execute time of the SharedTree for inserting a row and a column and removing them right away for a given number of times.
+				runBenchmark({
+					title: `Insert a column and a row and remove them right away ${count} times`,
+					tableSize,
+					cellValue,
+					operation: (table) => {
+						for (let i = 0; i < count; i++) {
+							const column = new Column({});
+							const row = new Row({ cells: {} });
+							table.insertColumn({ index: Math.floor(table.columns.length / 2), column });
+							table.insertRow({ index: Math.floor(table.rows.length / 2), row });
+							removeColumnAndCells(table, column);
+							table.removeRow(row);
+						}
+					},
+					maxBenchmarkDurationSeconds,
+				});
+
+				// Test the execute time of undoing insert a row and a column and removing them right away for a given number of times.
+				runUndoRedoBenchmark({
+					title: `Undo insert the middle column and row ${count} times`,
+					tableSize,
+					cellValue,
+					setupOperation: (table) => {
+						for (let i = 0; i < count; i++) {
+							const column = new Column({});
+							const row = new Row({ cells: {} });
+							table.insertColumn({ index: Math.floor(table.columns.length / 2), column });
+							table.insertRow({ index: Math.floor(table.rows.length / 2), row });
+							removeColumnAndCells(table, column);
+							table.removeRow(row);
+						}
+					},
+					stackOperation: (undoRedoManager) => {
+						for (let i = 0; i < count; i++) {
+							// Undo remove row
+							undoRedoManager.undo();
+							// Undo remove column
+							undoRedoManager.undo();
+							// Undo insert row
+							undoRedoManager.undo();
+							// Undo insert column
+							undoRedoManager.undo();
+						}
+						assert(!undoRedoManager.canUndo);
+					},
+					maxBenchmarkDurationSeconds,
+				});
+				// Test the execute time of the SharedTree for redoing an insert row and a column and removing them right away for a given number of times.
+				runUndoRedoBenchmark({
+					title: `Redo insert the middle column and row ${count} times`,
+					tableSize,
+					cellValue,
+					setupOperation: (table, undoRedoManager) => {
+						for (let i = 0; i < count; i++) {
+							const column = new Column({});
+							const row = new Row({ cells: {} });
+							table.insertColumn({ index: Math.floor(table.columns.length / 2), column });
+							table.insertRow({ index: Math.floor(table.rows.length / 2), row });
+							removeColumnAndCells(table, column);
+							table.removeRow(row);
+						}
+						for (let i = 0; i < count; i++) {
+							undoRedoManager.undo();
```

Comment:
Nit: I liked the comments you added in the test above explaining what each undo call is doing. Would you mind adding them here as well (and anywhere else we do multiple undo/redos)? 😁